download/tests/read-proxy-env.rs

```diff
@@ -0,0 +1,50 @@
+#![cfg(feature = "reqwest-backend")]
```

Comment:
Did this bug only affect one backend?

Reply:
Yes, cURL has its own proxy detection logic (which `env_proxy` emulates).